test/parse.jl

```diff
@@ -863,3 +863,6 @@ end
 
 @test QualifiedStringMacro.SubModule.x"" === 1
 @test QualifiedStringMacro.SubModule.y`` === 2
+
+..(x,y) = x + y
```

Comment:
I would use `let ..(x,y) = x+y ...`, or just test the result of `parse` rather than execute the definition.